KubeArmor/main.go

```diff
@@ -57,6 +61,28 @@ func main() {
 		kg.Err(err.Error())
 		return
 	}
+	if cfg.GlobalCfg.EnablePPROF {
+		go func() {
+			mux := http.NewServeMux()
+
+			mux.HandleFunc("/debug/pprof/", pprof.Index)
+			mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+
+			server := &http.Server{
+				Addr:              "0.0.0.0:" + cfg.GlobalCfg.PPROF,
+				Handler:           mux,
+				ReadHeaderTimeout: 10 * time.Second,
+			}
+
+			if err := server.ListenAndServe(); err != nil {
+				kg.Err(err.Error() + "\nCould not expose a pprof server for debugging")
+			} else {
+				fmt.Println("pprof server is running on http://0.0.0.0:" + cfg.GlobalCfg.PPROF)
+			}
```

Comment:
```suggestion
			kg.Printf("pprof server is running on http://0.0.0.0:" + cfg.GlobalCfg.PPROF)
			if err := server.ListenAndServe(); err != nil {
				kg.Err(err.Error() + "\nCould not expose a pprof server for debugging")
			}
```
Server and Listen is blocking function call, We would never log pprof server is running if we add it in else case. Also use our custom log package not fmt.